functions/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "noImplicitReturns": true,
     "outDir": "lib",
     "sourceMap": true,
-    "target": "es6"
+    "target": "es6",
+    "types": ["node"]
```

Comment:
See https://github.com/Microsoft/TypeScript/issues/13992
Without this, the ts compiler traced up to the parent directory and processed the node_modules directory in the parent, and caused weird errors.


```
> tsc

../node_modules/@types/history/DOMUtils.d.ts(2,78): error TS2304: Cannot find name 'EventListenerOrEventListenerObject'.
../node_modules/@types/history/DOMUtils.d.ts(3,81): error TS2304: Cannot find name 'EventListenerOrEventListenerObject'.
../node_modules/@types/reach__router/index.d.ts(9,30): error TS2304: Cannot find name 'Window'.
```